packages/lib/src/components/BacsDD/components/BacsResult.tsx

```diff
@@ -7,15 +7,15 @@ import './BacsResult.scss';
 import useImage from '../../../core/Context/useImage';
 
 const BacsResult = props => {
-    const { i18n, loadingContext } = useCoreContext();
+    const { i18n } = useCoreContext();
     const getImage = useImage();
     const { url, paymentMethodType } = props;
 
     return (
         <Voucher
             paymentMethodType={paymentMethodType}
             introduction={i18n.get('bacs.result.introduction')}
-            imageUrl={getImage({ loadingContext })(paymentMethodType)}
+            imageUrl={getImage({})(paymentMethodType)}
```

Comment:
Do you mean removing the object, or change the return on getImage() to not be a function?
I tried to play around with the second one, but there were places were we use getImage to get the function configured for the whole component and then reuse it in specific places.

Reply:
I meant the first part, passing an empty object

Reply:
Going to fix that

Reply:
Done.
Also played around with making that the configurations get passed as part of `useImage`:

```ts
const getImage = useImage();
getImage({imageFolder: 'components/'})('success')
```

Would become:

```ts
const getImage = useImage({imageFolder: 'components/'});
getImage('success')
```

But I'm not sure if it makes sense to be a configuration of useImage, because it can be different in the same component, ie:

```ts
getImage({ extension: 'gif', imageFolder: 'components/' })('success')
```

In this case `gif` is more part of `success` icon and the whole component itself, because another image that get's loaded here might be an `svg`.

So in the future we might want to move this configurations to `Resources.getImage` and make the props as a second argument. Honestly this whole thing should be much simpler, but it's a bit over engineered at this point and removing complexity is harder.